medmodels/medrecord/medrecord.py

```diff
@@ -566,41 +569,15 @@ def edges_connecting(
                 (target_node if isinstance(target_node, list) else [target_node]),
             )
 
-    def add_node(
-        self,
-        node: NodeIndex,
-        attributes: AttributesInput,
-        group: Optional[Group] = None,
-    ) -> None:
-        """Adds a node with specified attributes to the MedRecord instance. Optionally adds the node to a group.
-
-        Args:
-            node (NodeIndex): The index of the node to add.
-            attributes (Attributes): A dictionary of the node's attributes.
-            group (Optional[Group]): The name of the group to add the node to, optional.
-
-        Returns:
-            None
-        """
-        self._medrecord.add_node(node, attributes)
-
-        if group is None:
-            return
-
-        if not self.contains_group(group):
-            self.add_group(group)
-
-        self.add_node_to_group(group, node)
-
     @overload
-    def remove_node(self, node: NodeIndex) -> Attributes: ...
+    def remove_nodes(self, node: NodeIndex) -> Attributes: ...
 
     @overload
-    def remove_node(
+    def remove_nodes(
         self, node: Union[NodeIndexInputList, NodeOperation]
     ) -> Dict[NodeIndex, Attributes]: ...
 
-    def remove_node(
+    def remove_nodes(
         self, node: Union[NodeIndex, NodeIndexInputList, NodeOperation]
```

Comment:
Should we rename the variable to `nodes` ?

Reply:
yes